_posts/2019-12-20-performance-now-conference-2019.md

```diff
@@ -0,0 +1,96 @@
+
+Over the years I’ve made use of many tools to gain a better understanding of site performance - here are a few of my favourites that are free, easy to use, and provide a wealth of advice and insights :
+
+* [Sitespeed.io](https://www.sitespeed.io): Not mentioned at the conference but one of my favourites. Sitespeed.io is a set of Open Source tools that makes it easy to monitor and measure the performance of your web site. Sitespeed.io is the complete toolbox to test and  monitor your performance or checkout how your competition is doing.
+
+* [Webpagetest](https://www.webpagetest.org): Simple to use and lots of options to play with. WebPagetest is used for measuring and analysing the performance of web pages. A test can be kicked off from a variety of locations with a range of different browsers and test configurations.
+
+* [Pagespeed Insights](https://developers.google.com/speed/pagespeed/insights/): PageSpeed Insights (PSI) reports on the performance of a page on both mobile and desktop devices, and provides suggestions on how that page may be improved. PSI provides both lab and field data about a page. Lab data is useful for debugging performance issues, as it is collected in a controlled environment.
+
+While quick one-off tests can be executed against your site with relative ease, with a little more work it’s possible to integrate such tests into your continuous integration pipeline to provide feedback earlier in the development lifecycle.
+
+
+### Takeaway 3 - Don’t be a Performance Cop
+Once equipped with the knowledge, tools, and data necessary to improve site performance, there is still a very big challenge to managing and maintaining performance especially in a fast-paced environment where multiple teams can be releasing changes into production several times throughout the day.
+
+![Performance Budgets](/images/performance-now-conf-03.png)
+ 
+> “Everyone who touches a page should care about the performance of that page” - Tammy Everts
+
+Tammy Everts talk entitled [“The 7 habits of highly effective performance teams”](https://www.youtube.com/watch?v=SE0HhF4TO0Q) shared tips and best practice gleaned from conversations with companies leading the way in web performance. These companies had one thing in common - they had a strong culture of performance. 
+
+The 7 habits covered in the talk were as follows:
+
+* 1 - Performance is a marathon, not a sprint
```

Comment:
If this is meant to be an ordered list, use [markdown format](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#lists)

Reply:
fixed